### PR TITLE
Updated to 0.2.1

### PR DESCRIPTION
**Pull request by mariusvniekerk**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -44,3 +44,4 @@ about:
 extra:
   recipe-maintainers:
     - mariusvniekerk
+    - ericdill
```
Comment (jakirkham) [inline recipe/meta.yaml:47]: Is this ok with you, @ericdill?

Comment (ericdill) [inline recipe/meta.yaml:47]: Yup. I'll post on that issue giving Marius the green light to add me to whatever he sees fit

Comment (ericdill) [inline recipe/meta.yaml:47]: Thanks for checking :+1:

Comment (jakirkham) [inline recipe/meta.yaml:47]: Of course. 😄

**Pull request merged**